staging/src/k8s.io/kubectl/pkg/cmd/portforward/portforward.go

```diff
@@ -225,6 +226,74 @@ func convertPodNamedPortToNumber(ports []string, pod corev1.Pod) ([]string, erro
 	return converted, nil
 }
 
+// checkUDPPortInService port returns error if remote port in Service is a UDP port
```

Comment:
For me the word `port` after checkUDPPortInService sounds strange to me. I'm not sure, but I think this makes more sense:
```diff
- // checkUDPPortInService port returns error if remote port in Service is a UDP port
+ // checkUDPPortInService returns an error if remote port in Pod is an UDP port
```